### PR TITLE
Fixed Sphinx errors

### PR DESCRIPTION
**Pull request by heuer**
I got some errors while building the docs with Sphinx. 
These changes fix them.
Additionally, a repetition in "barcode.rst" was fixed.

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ㊙️ python-barcode's documentation
-=================================
+===================================
```
Comment (WhyNotHugo) [inline docs/source/index.rst]: Two extra characters here?

Comment (heuer) [reply docs/source/index.rst]: I get the following error (Py 2.7)
```
index.rst:2:Title underline too short.

\u3299\ufe0f python-barcode's documentation
=================================
```

and (Py 3.7):

```
index.rst:2:Title underline too short.

㊙️ python-barcode's documentation
=================================
```

Comment (WhyNotHugo) [reply docs/source/index.rst]: Huh, odd. I missed that warning before. Makes sense then.

**Comment by WhyNotHugo** [comment]
Merged! Thanks for contributing! :)

**Pull request merged**